packages/wifi_basic/ios/Classes/SwiftWifiBasicPlugin.swift

```diff
@@ -0,0 +1,65 @@
+import Flutter
+import UIKit
+
+public class SwiftWifiBasicPlugin: NSObject, FlutterPlugin {
```

Comment:
@DominikStarke Can you review this? esp. handling of `isEnabled` (lifted code from SO!) - tested with iOS - iOS 15 simlulator (returns true), and iOS 12 device (it is working), will soon test on iOS 15 too - will let you know.

Reply:
Looks like it always returns true in iOS 15.

So may be wont go ahead with this unofficial API.

@DominikStarke  Do you have any views on what best-case API to call if iOS does not allow it? Currently in `wifi_iot` - it checks if `currentSSID != null` - other forums suggest using `SCNetworkReachability` (as used by `connectivity_plus` plugin). 

Both suggest that the mobile is "connected" to a WifiNetwork and not if the hw/service is "enabled" - but looks like this is the best we can achieve. My views are we should use `SCNetworkReachability` as it does not require any additional permission from user. Le me know your view.